package.json

```diff
@@ -55,6 +55,7 @@
     "appId": "com.wulkano.kap",
     "files": [
       "**/*",
+      "dist",
```

Comment:
Wasted a lot of time on this error message:

> Error: Application entry file "dist/main/index.js" does not exist. Seems like a wrong configuration.

Turns out the `dist` directory was not copied over for a reason. I'll dig more into it later, but for now, it's easy enough to manually add it here. Seems like an `electron-builder` issue.